core/js/setupchecks.js

```diff
@@ -423,6 +423,24 @@
 							type: OC.SetupChecks.MESSAGE_TYPE_INFO
 						})
 					}
+					if (!data.isImagickEnabled) {
+						messages.push({
+							msg: t(
+								'core',
+								'Imagick is not enabled although the theming app is. For Favicon generation to work correctly, you need to install the imagick PHP extension.'
```

Comment:
```suggestion
								'The PHP module "imagick" is not enabled although the theming app is. For favicon generation to work correctly, you need to install and enable this module.'
```
Just for aligning with other messages 🙂.

Reply:
lgtm :)